kiwi/system/root_init.py

```diff
@@ -84,6 +85,8 @@ def create(self):
             data.sync_data(
                 options=['-a', '--ignore-existing']
             )
+            if Defaults.is_buildservice_worker():
+                copy('/.buildenv', self.root_dir)
```

Comment:
Thinking twice about this, your former implementation was totally correct , sorry for the confusion. The detection if we are a bs worker is based on the presence of .buildenv, thus the copy of that file is safe and makes complete sense.

So hope you don't mind but I think we should revert the last commit

Once done I only have only issue. With the copy statement in place we would have two places in the code which deals with the .buildenv file name. I suggest we create a method in the defaults area e.g:

```
@classmethod
def get_buildservice_env_name(self):
    """
    The base name of the environment file in a buildservice worker
    """
    return '.buildenv'
```

and use that in the is_buildservice_worker() method and in the copy() call

Thoughts ?